### DISCUSSION TITLE
XRD texture and UXD format file

### DISCUSSION THREAD

**Discussion by lyg1232** [mtex-toolbox/mtex#1470] [Ask Anything]
Hi，Here is my xrd raw data, but as per mtex's documentation, the data cannot be imported correctly，can you help me with this problem?Thank you!
![2](https://user-images.githubusercontent.com/81472625/175949923-20d69b56-d957-4ca7-923d-2ec19b0a6f11.png)
![3](https://user-images.githubusercontent.com/81472625/175950287-fabd5c38-76ca-4f72-a0ca-9d0e1e8e1dcc.png)

**Comment by kilir** [comment]
Hi,
are you sure that fname points to the correct place?
Cheers,
Rüdiger
